include/cantera/thermo/MixtureFugacityTP.h

```diff
@@ -1,577 +1,572 @@
+     */
+    int phaseState(bool checkState = false) const;
+
+    //! Return the value of the density at the liquid spinodal point (on the
+    //! liquid side) for the current temperature.
+    /*!
+     * @returns the density with units of kg m-3
+     */
+    virtual doublereal densSpinodalLiquid() const;
+
+    //! Return the value of the density at the gas spinodal point (on the gas
+    //! side) for the current temperature.
+    /*!
+     * @returns the density with units of kg m-3
+     */
+    virtual doublereal densSpinodalGas() const;
+
+public:
+    //! Calculate the saturation pressure at the current mixture content for the
+    //! given temperature
+    /*!
+     * @param TKelvin         (input) Temperature (Kelvin)
+     * @param molarVolGas     (return) Molar volume of the gas
+     * @param molarVolLiquid  (return) Molar volume of the liquid
+     * @returns the saturation pressure at the given temperature
+     */
+    doublereal calculatePsat(doublereal TKelvin, doublereal& molarVolGas,
+                             doublereal& molarVolLiquid);
+
+public:
+    //! Calculate the saturation pressure at the current mixture content for the
+    //! given temperature
+    /*!
+     * @param TKelvin   Temperature (Kelvin)
+     * @return          The saturation pressure at the given temperature
+     */
+    virtual doublereal satPressure(doublereal TKelvin);
+	virtual void setToEquilState(const doublereal* lambda_RT);
+    virtual void getActivityConcentrations(double* c) const;
+
+protected:
+    //! Calculate the pressure given the temperature and the molar volume
+    /*!
+     * @param   TKelvin   temperature in kelvin
+     * @param   molarVol  molar volume ( m3/kmol)
+     * @returns the pressure.
+     */
+    virtual doublereal pressureCalc(doublereal TKelvin, doublereal molarVol) const;
+
+    //! Calculate the pressure and the pressure derivative given the temperature
+    //! and the molar volume
+    /*!
+     *  Temperature and mole number are held constant
+     *
+     * @param   TKelvin   temperature in kelvin
+     * @param   molarVol  molar volume ( m3/kmol)
+     * @param   presCalc  Returns the pressure.
+     * @returns the derivative of the pressure wrt the molar volume
+     */
+    virtual doublereal dpdVCalc(doublereal TKelvin, doublereal molarVol, doublereal& presCalc) const;
+
+    virtual void updateMixingExpressions();
+
+
+    //! Solve the cubic equation of state
+    /*!
+     * 
+     * Returns the number of solutions found. If it only finds the liquid
+     * branch solution, it will return a -1 or a -2 instead of 1 or 2.  If it
+     * returns 0, then there is an error.
+     * The cubic equation is solved using Nickall's method (Ref: The Mathematical Gazette(1993), 77(November), 354�359, https://www.jstor.org/stable/3619777)
+     */
+    int solveCubic(double T, double pres, double a, double b, 
+                                        double aAlpha, double Vroot[3], double an,

```

Comment:
The formatting here is weird to me, should the `double aAlpha` line up under the `double T`?

Reply:
My apologies about all these indentation errors. I have fixed this one and all others you mentioned below.